src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -15,7 +15,7 @@
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.paginator import OffsetPaginator
 from sentry.api.serializers import serialize, CombinedRuleSerializer
-from sentry.incidents.endpoints.serializers import AlertRuleSerializer
+from sentry.incidents.endpoints.serializers import UnifiedAlertRuleSerializer
```

Comment:
This refactor (https://github.com/getsentry/sentry/pull/16546) removes the UnifiedAlertRuleSerializer

Reply:
once it's merged, we can leave this as-is (using AlertRuleSerializer)